fastlane_core/lib/fastlane_core/fastlane_pty.rb

```diff
@@ -10,48 +10,78 @@ def exit_status
 
 module FastlaneCore
   class FastlanePtyError < StandardError
-    attr_reader :exit_status
-    def initialize(e, exit_status)
+    attr_reader :exit_status, :process_status
```

Comment:
I decided to expose the status, as it holds more information about the crash when it happens (e.g. the signal number, the pid). It's not fully necessary.